tx/textual/valuerenderer/int_test.go

```diff
@@ -64,4 +67,7 @@ func checkNumberTest(t *testing.T, r valuerenderer.ValueRenderer, pv protoreflec
 	require.Equal(t, false, screens[0].Expert)
 
 	require.Equal(t, expected, screens[0].Text)
+	value, err := r.Parse(context.Background(), screens)
+	require.NoError(t, err)
+	require.Equal(t, strings.Replace(screens[0].Text, "'", "", -1), value.String())
```

Comment:
```suggestion
	require.Equal(t, pv.String(), value.String())
```

Reply:
This will break a few cases:
for ex: we have a case `int.json` file as below

"001" -> format -> "1"  
"1" -> parse -> "1" ("1" != "001")

Reply:
I see. let's check equality using math.Int then:

```go
i1 := math.NewIntFromString(pv.String())
i2 := math.NewIntFromString(value.String())
require.True(t, i1.Eq(i2))
```